test/modules/DirectMethodSender/Program.cs

```diff
@@ -141,5 +110,31 @@ public static bool IsTestTimeUp(DateTime testStartAt)
         {
```

Comment:
Suggest to rename as "ContinueTesting"

Reply:
I think IsTestTimeUp() is more explicit of what it is doing.

Reply:
Check with others.